lib-src/libnyquist/nyx.c

```diff
@@ -1002,22 +1002,6 @@ int nyx_get_audio(nyx_audio_callback callback, void *userdata)
       goto finish;
    }
 
-   // if LEN is set, we will return LEN samples per channel. If LEN is
-   // unbound, we will compute samples until every channel has terminated
-   // that the samples per channel will match the last termination time,
-   // i.e. it could result in a partial block at the end.
-   if (nyx_input_length == 0) {
-      LVAL val = getvalue(xlenter("LEN"));
-      if (val != s_unbound) {
-         if (ntype(val) == FLONUM) {
-            nyx_input_length = (int64_t) getflonum(val);
-         }
-         else if (ntype(val) == FIXNUM) {
-            nyx_input_length = (int64_t) getfixnum(val);
-         }
-      }
-   }
```

Comment:
The comment was added with cd79f1099b0ca59167dfd895e0d06f99123b44e8 but the actual code goes back to the start of the Git repo.  So this commit is not just restoring behavior "as before" cd79f1099b0ca59167dfd895e0d06f99123b44e8.

That is not a criticism.  I'd just like to understand this more deeply.